Contents/Code/__init__.py

```diff
@@ -152,14 +152,14 @@ def Update(self, metadata, media, lang, force, movie):
 
                 if len(series['art']['thumb']):
                     for art in series['art']['thumb']:
-                        episodeObj.thumbs[art['url']] = Proxy.Media(HTTP.Request(art['url']).content, art['index'])
+                        episodeObj.thumbs[art['url']] = Proxy.Media(HTTP.Request("http://{host}:{port}{relativeURL}".format(host=Prefs['Hostname'], port=Prefs['Port'], relativeURL=art['url'])).content, art['index'])
```

Comment:
this could be: `"http://%s:%d%s" % (Prefs['Hostname'], Prefs['Port'], art['url'])`